backgroundcheck_replay/tests/backgroundcheck_test.go

```diff
@@ -77,14 +77,14 @@ Run the tests in the test directory (`go test`).
 If the Workflow Definition and the Event History are incompatible then the test fails.
 */
 
-// TestReplayWorkflowHistoryFromFile tests for Event History compatability.
+// TestReplayWorkflowHistoryFromFile tests for Event History compatibility.
 func (s *UnitTestSuite) TestReplayWorkflowHistoryFromFile() {
 	// Create a new Replayer
 	replayer := worker.NewWorkflowReplayer()
 	// Register the Workflow with the Replayer
 	replayer.RegisterWorkflow(workflows.BackgroundCheck)
 	// Compare the current Workflow code against the existing Event History
-	err := replayer.ReplayWorkflowHistoryFromJSONFile(nil, "backgroundcheck_workflow_event_history.json")
+	err := replayer.ReplayWorkflowHistoryFromJSONFile(nil, "backgroundcheck_workflow_history.json")
```

Comment:
Revert this I think, because the CLI command in the other sections actually specify the old filename:

```
```shell
temporal workflow show \
 --workflow-id backgroundcheck_workflow \
 --namespace backgroundcheck_namespace \
 --output json > backgroundcheck_workflow_event_history.json
```